integration_tests/cairo_files/blake.small.cairo

```diff
@@ -0,0 +1,124 @@
+// inspired from the blake integration tests in the lambdaclass cairo-vm codebase
+
+%builtins range_check bitwise
```

Comment:
how can use use layout small for this file ? Bitwise is not in the small layout right?